website_sale_product_description/views/product_template_views.xml

```diff
@@ -1,18 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <data>
-
     <record model="ir.ui.view" id="product_template_form_view">
         <field name="name">product.template.website.sale.product.description.form</field>
         <field name="model">product.template</field>
         <field name="inherit_id" ref="product.product_template_form_view"/>
         <field name="arch" type="xml">
             <group name="email_template_and_project" position="after">
-                <separator string="Description for Website"/>
-                <field name="description_website" placeholder="note to be displayed on website..."/>
+                <group>
```

Comment:
* Currently it looks like this

  ![image](https://user-images.githubusercontent.com/7593953/41599733-6d3396a2-73aa-11e8-8317-7b8903cb16fd.png)

* Please make it look like this one 

  ![image](https://user-images.githubusercontent.com/7593953/41600038-31afff3e-73ab-11e8-98f1-93a7d01c97ee.png)

   - dont use separator, use group string instead
   - all the descriptions of the product are now in the Notes page, please move it to this section
   - Dont show the field label.

```xml
<group name="description" position="inside">
    <group string="Description for Website">
        <field name="description_website" nolabel="1" placeholder="Note to be displayed on e-commerce..."/>
    </group>
</group>
```